pkg/datapath/linux/node.go

```diff
@@ -509,9 +509,15 @@ func upsertIPsecLog(err error, spec string, loc, rem *net.IPNet, spi uint8) {
 	})
 	if err != nil {
 		scopedLog.WithError(err).Error("IPsec enable failed")
+		return fmt.Errorf("create endpoint: %s (local_ip=%s, rem=%s, spi=%d): %w",
+			spec,
+			loc.String(),
+			rem.String(),
+			spi, err)
 	} else {
```

Comment:
I think we should avoid the custom formatting (`local_ip=...`) and instead make the error msg itself more naturally flow:

```
return fmt.Errorf("failed to create endpoint for %q with local IP %q and remote IP %q: %w", spec, loc, rem)
```

Reply:
Not sure what "create endpoint" is referring to here though

Reply:
Also, if we are already going to log when this error occurs, then we don't need to stuff the error msg with the same context again. It would also be good to use the newly created error within the log msg, i.e. `WithError()`

Reply:
I was in between two priorities in many of these:

* keeping the error messages as terse/short as possible as to avoid excessively long errors once they've made their way up the stack.
* Providing enough context info such that users aren't left scratching their heads about why exactly the error occured.